src/itest/kotlin/io/kotest/provided/ProjectConfig.kt

```diff
@@ -19,18 +19,22 @@ import nl.lifely.zac.itest.config.ItestConfiguration.PRODUCT_AANVRAAG_TYPE
 import nl.lifely.zac.itest.config.ItestConfiguration.ZAAKTYPE_MELDING_KLEIN_EVENEMENT_IDENTIFICATIE
 import nl.lifely.zac.itest.config.ItestConfiguration.ZAAKTYPE_MELDING_KLEIN_EVENEMENT_UUID
 import nl.lifely.zac.itest.config.ItestConfiguration.ZAC_API_URI
+import nl.lifely.zac.itest.config.ItestConfiguration.ZAC_MANAGEMENT_URI
+import org.awaitility.kotlin.await
 import org.slf4j.Logger
 import org.testcontainers.containers.ComposeContainer
 import org.testcontainers.containers.ContainerLaunchException
 import org.testcontainers.containers.output.Slf4jLogConsumer
 import org.testcontainers.containers.wait.strategy.Wait
 import java.io.File
 import java.time.Duration
+import java.util.concurrent.TimeUnit
 
 private val logger = KotlinLogging.logger {}
 
 object ProjectConfig : AbstractProjectConfig() {
     private const val THREE_MINUTES = 3L
+    private const val TEN_SECONDS = 10L
```

Comment:
don't want to be picky, but the value does not have a unit, so this is just `TEN`, and it's used in combination with the seconds unit below.
You could create it as a tuple (or the like) so that it also contains the unit.

Similar to the `THREE_MINUTES` constant ofc

Reply:
Maybe make these both `Durations`?

Reply:
hmm.  if you do ` private val THREE_MINUTES = Duration.ofMinutes(3)` Detekt will still complain that '3' is a magic number and suggest to move it into a constant.  you can suppress this warning of course. :-)

Reply:
ps: 'await.atMost' takes a long, not a duration. Which internally they convert into a Duration.. ;-)